apps/expo/src/app/userprofile/index.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import React from "react";
+import {
+  Button,
+  Image,
+  ImageBackground,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Stack, useRouter } from "expo-router";
+
+const Index = () => {
+  return (
+    <SafeAreaView className="bg-[#000000]">
+      {/* Changes page title visible on the header */}
+      <Stack.Screen
+        options={{
+          headerTitle: (props) => (
+            <View className="container">
+              <Image
+                className="mx-auto"
+                source={require("../../../assets/Logo-mini.png")}
+              />
+            </View>
+          ),
+        }}
+      />
+      <Image source={require("../../../assets/headerBg.png")} />
+      <View className="container flex h-full w-full flex-row space-x-10 bg-black">
+        <Image
+          style={{
```

Comment:
we should try to make this style with taildwind

Reply:
you can do it in next PR.

Reply:
> we should try to make this style with taildwind

> you can do it in next PR.

Got it, will fix it along side the "icons" implementation